apps/srp.c

```diff
@@ -553,6 +553,8 @@ int srp_main(int argc, char **argv)
             user = *(argv++);
         else {
```

Comment:
I would suggest that since we are here it would be appropriate to bring it to compliance with coding style guide. If mean one should add {} to *above* if statement.